<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Vikraman</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGAL Regular Triangulation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear"/>
   <addaction name="actionLoadPoints"/>
   <addaction name="actionSavePoints"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Visualization Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInsertPoint"/>
   <addaction name="actionInsertLine"/>
   <addaction name="separator"/>
   <addaction name="actionShowRegular"/>
   <addaction name="actionShowPowerdiagram"/>
   <addaction name="actionShowConvexHull"/>
   <addaction name="actionShowMoldLpp"/>
   <addaction name="actionShowIntersection"/>
   <addaction name="separator"/>
   <addaction name="actionRecenter"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="actionLoadPoints"/>
    <addaction name="actionSavePoints"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionInsertRandomPoints"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionInsertPoint"/>
    <addaction name="actionInsertLine"/>
    <addaction name="separator"/>
    <addaction name="actionShowRegular"/>
    <addaction name="actionShowPowerdiagram"/>
    <addaction name="actionShowConvexHull"/>
    <addaction name="actionShowMoldLpp"/>
    <addaction name="actionShowIntersection"/>
    <addaction name="separator"/>
    <addaction name="actionRecenter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutCGAL">
   <property name="text">
    <string>About &amp;CGAL</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInsertRandomPoints">
   <property name="text">
    <string>&amp;Insert random points</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionInsertPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/Input/inputPoint.png</normaloff>:/cgal/Input/inputPoint.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Insert weighted point</string>
   </property>
   <property name="toolTip">
    <string>Insert weighted point</string>
   </property>
   <property name="statusTip">
    <string>Left: Insert vtx</string>
   </property>
  </action>
  <action name="actionInsertLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/Input.qrc">
    <normaloff>:/cgal/Input/inputPolyline.png</normaloff>:/cgal/Input/inputPolyline.png</iconset>
   </property>
   <property name="text">
    <string>Input Line</string>
   </property>
   <property name="toolTip">
    <string>Input Lines</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/fileToolbar/fileNew.png</normaloff>:/cgal/fileToolbar/fileNew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionShowPowerdiagram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/view/Voronoi_diagram.png</normaloff>:/cgal/view/Voronoi_diagram.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Voronoi Diagram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionShowRegular">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
        <normaloff>:/cgal/view/Triangulation.png</normaloff>:/cgal/view/Triangulation.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Regular Triangulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionShowConvexHull">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
        <normaloff>:/cgal/view/Convex_hull.png</normaloff>:/cgal/view/Convex_hull.png</iconset>
   </property>
   <property name="text">
    <string>Show Convex &amp;Hull</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionShowMoldLpp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/view/Mold_lpp.png</normaloff>:/cgal/view/Mold_lpp.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Mold Lpp</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionShowIntersection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/View.qrc">
    <normaloff>:/cgal/view/Intersection.png</normaloff>:/cgal/view/Intersection.png</iconset>
   </property>
   <property name="text">
    <string>Show I&amp;ntersection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="toolTip">
    <string>Show Intersection Points</string>
   </property>
  </action>
  <action name="actionLoadPoints">
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/fileToolbar/fileOpen.png</normaloff>:/cgal/fileToolbar/fileOpen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Points...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSavePoints">
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/fileToolbar/fileSave.png</normaloff>:/cgal/fileToolbar/fileSave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Points...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRecenter">
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/Input/zoom-best-fit</normaloff>:/cgal/Input/zoom-best-fit</iconset>
   </property>
   <property name="text">
    <string>Re&amp;center the viewport</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainwindow.qrc"/>
  <include location="icons/File.qrc"/>
  <include location="resources/CGAL.qrc"/>
  <include location="icons/View.qrc"/>
  <include location="icons/Input.qrc"/>
 </resources>
 <connections/>
</ui>
